cogs/commands/dalle.py

```diff
@@ -54,13 +74,75 @@ async def get_image(self, url):
             async with session.get(url) as response:
                 if response.status == 200:
                     logging.info("successfully got image")
-                    return discord.File(
-                        BytesIO(await response.read()), filename="image.png"
-                    )
+                    return BytesIO(await response.read())
                 else:
                     logging.info("failed to get image")
                     return None
 
+    async def generate_variant(self, image):
```

Comment:
This is static so can be moved outside the class (or mark with `@staticmethod`) and remove `self`